nilearn/decoding/__init__.py

```diff
@@ -4,3 +4,9 @@
 
 from .searchlight import SearchLight
 from .space_net import SpaceNetClassifier, SpaceNetRegressor
+from .searchlight import search_light
+from .fista import mfista
+from .space_net_solvers import tvl1_solver
+
+__all__ = ["SearchLight", "SpaceNetRegressor", "SpaceNetClassifier",
+           "search_light", "mfista", "tvl1_solver"]
```

Comment:
I don't think that we want to expose any of the functions here. Just the objects.